plugins/chain_api_plugin/chain.swagger.yaml

```diff
@@ -724,3 +724,102 @@ paths:
+    post:
+      description: This method expects a transaction in JSON format and will attempt to apply it to the blockchain. It supports returning the full trace of a failed transaction and automatic nodeos-mediated retry if it is enabled on the node. When transaction retry is enabled on an API node, it will monitor incoming API transactions and ensure they are resubmitted additional times into the P2P network until they expire or are included in a block. Warning, full failure traces are now returned by default instead of exceptions. Be careful to not confuse a returned trace as an indication of speculative execution success. Verify 'receipt' and 'except' fields of the returned trace.
+      operationId: send_transaction2
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                return_failure_trace:
+                  type: boolean
+                  description: If true then embed transaction exceptions into the returned transaction trace.
+                retry_trx:
+                  type: boolean
+                  description: If true then request transaction retry on validated transaction.
+                retry_trx_num_blocks:
+                  type: integer
+                  description: If retry_trx is true, report trace at specified blocks from executed or lib if not specified.
+                transaction:
+                  type: object
+                  properties:
+                    signatures:
+                      type: array
+                      description: array of signatures required to authorize transaction.
+                      items:
+                        $ref: "https://docs.eosnetwork.com/openapi/v2.0/Signature.yaml"
+                    compression:
+                      type: boolean
+                      description: Compression used, usually false
+                    packed_context_free_data:
+                      type: string
+                      description: json to hex
+                    packed_trx:
+                      type: string
+                      description: Transaction object json to hex
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                description: Returns Nothing
+
+  /compute_transaction:
+    post:
+      description: Transactions sent to this endpoint will be executed and create a transaction trace, including resource usage, and then will revert all state changes and will not contribute to the subjective billing for the account. If the transaction has signatures, they will be processed, but any failures will be ignored. Transactions which fail will always include the transaction failure trace. Warning, users with exposed nodes who have enabled the compute_transaction endpoint should implement some sort of throttling to protect from Denial of Service attacks.
```

Comment:
consider starting description with:
"Executes specified transaction and creates a transaction trace...." 

convert future tense to present tense
"... and then will revert all" -> "... and then reverts all"
"and will not contribute" -> "but do not contribute"
"they will be processed" -> "they are processed"
etc.